### DISCUSSION TITLE
Where can I find the release by version

### DISCUSSION THREAD

**Discussion by useruser456** [firebase/firebase-android-sdk#3866] [Q&A] [answered]
Hello,

I'm trying to find the source code for com.google.firebase:firebase-bom:28.4.0

I don't see any tags that show that. Can you help me ?

Br

**Comment by daymxn** [comment, marked as answer]
> I'm trying to find the source code for com.google.firebase:firebase-bom:28.4.0

Our BOM generation can be found [here](https://github.com/firebase/firebase-android-sdk/blob/master/buildSrc/src/main/java/com/google/firebase/gradle/bomgenerator/BomGeneratorTask.java).
You're not finding tags for it because BOM generation is a `core` plugin, it's not a separate module and doesn't have a unique team behind it (that is, a team designated to specifically working on BOM generation *only*).

If that doesn't quite answer your question, reopen this and I can help you out:)
